src/components/Calculator.js

```diff
@@ -1,48 +1,72 @@
 import React, { Component } from 'react';
+import calculate from '../logic/calculate';
 
 // eslint-disable-next-line
 class Calculator extends Component {
+  constructor() {
```

Comment:
- [ ] You did great job. However, you have some errors showing up. In order to make your project approve. Your all buttons are must be functional, and clickable. You can watch the video and see the errors. 

https://user-images.githubusercontent.com/69011963/198722541-b0719be2-00be-4c0c-903a-a4d30cb30834.mov